Dockerfile

```diff
@@ -15,9 +16,12 @@ COPY pkg ./pkg
 
 RUN make build
 
+FROM quay.io/openshift/origin-cli:latest as origincli
```

Comment:
@jcantrill / @ewolinetz Do you know if this needs an amendment in `ocp-build-data` for ART?

Reply:
yes I believe it does

Reply:
Is this how this works, FROM layers are munged together?

Reply:
No FROM layers build different overlays. Directives in between are optional. Think of having multiple FROM directives on the top like branches that you want to merge by cherry-pick in a single final overlay.